pymba/vimba.py

```diff
@@ -64,17 +64,15 @@ def _getInterfaceInfos(self):
         """
         if self._interfaceInfos is None:
             # args
-            dummyInterfaceInfo = structs.VimbaInterfaceInfo()
             numFound = c_uint32(-1)
 
             # call once just to get the number of interfaces
             # Vimba DLL will return an error code
-            errorCode = VimbaDLL.interfacesList(byref(dummyInterfaceInfo),
+            errorCode = VimbaDLL.interfacesList(None,
```

Comment:
Please check if older versions of Vimba (< 1.5.0) also accept NULL as first parameter